extension/options/optionsView.jsx

```diff
@@ -498,3 +557,49 @@ const AboutSection = () => {
     </fieldset>
   );
 };
+
+const Tabs = ({
+  updateActiveTab
+}) => {
+  return (
+    <fieldset id="tabs">
```

Comment:
A fieldset isn't necessary here. This isn't really a form, it's just navigation.